synonymdev/bitkit-react-native
# feat: show failed backup warning

# Description

BitKit now remembers time of each backup attempt. If last attempt was more than 30 minutes ago and still no success, BitKit will start to show backup error message every 10 minutes.

---

## Type of change

New feature (non-breaking change which adds functionality)

## Tests
No test

## Screen shot / Video
Insert Relevant screenshot / Video share

![image](https://user-images.githubusercontent.com/155891/232524660-ef945dd8-4bd5-4472-a689-4384bcfce14c.png)

## QA Notes

- Turn device into airplate mode
- Change settings or add a few tags
- wait 30-40 minutes and then you should see backup failed warning

## Bitkit Version
